app/views/personal_details/new.html.erb

```diff
@@ -9,14 +9,14 @@
         </h1>
       </legend>
 
-      <%= f.govuk_text_field :title, label: { text: t('application_form.personal_details_section.title.label')}, width: 10 %>
-      <%= f.govuk_text_field :first_name, label: { text: t('application_form.personal_details_section.first_name.label')} %>
-      <%= f.govuk_text_field :preferred_name, label: { text: t('application_form.personal_details_section.preferred_name.label')} %>
-      <%= f.govuk_text_field :last_name, label: { text: t('application_form.personal_details_section.last_name.label')} %>
+      <%= f.govuk_text_field :title, label: { text: t('application_form.personal_details_section.title.label'), size: 's' }, width: 10 %>
+      <%= f.govuk_text_field :first_name, label: { text: t('application_form.personal_details_section.first_name.label'), size: 's' } %>
+      <%= f.govuk_text_field :preferred_name, label: { text: t('application_form.personal_details_section.preferred_name.label'), size: 's' } %>
```

Comment:
What's the motivation for the size change? Did people respond better to the bold text as labels?

Reply:
This design decision is not based on any user research (yet!). However, on pages such as the example below (where the form contains ‘sub’ labels for date inputs, alongside hint text and other guidance) it can be difficult to pick out the labels for each question.

One means of mitigating this issue might be to ask one question per page, but the working assumption is that this would make filling out each section – and thus the overall application – too onerous (the one thing we are asking for here is a role, not a job title, then organisation, then start date etc.). But this is just a best guess at this stage!

Without `small` modifier: 
![govuk-label](https://user-images.githubusercontent.com/813383/61627189-0bc42380-ac77-11e9-9609-669c4a2d9d45.png)

With `small` modifier: 
![govuk-label--s](https://user-images.githubusercontent.com/813383/61627187-0bc42380-ac77-11e9-9359-03712d0d3602.png)

With this situation in mind, it seems reasonable to then be consistent with how we style labels elsewhere on the service.

Reply:
> the working assumption is that this would make filling out each section – and thus the overall application – too onerous

This is interesting - didn't realise that was the assumption!

I think the GOV.UK research has generally shown lots of questions works out well - but I heartily agree we should research both to find out.

In the case here where we have no evidence either way, I'd always start with the GOV.UK standard unless research shows differently. I'm very happy to defer to you on design decisions like this, though.